ui/app/src/App.tsx

```diff
@@ -19,44 +56,38 @@ function App() {
       >
         <ConnectButton />
       </div>
+
+      {/* Transaction Status Display */}
       <div
         style={{
           display: "flex",
           alignItems: "center",
           justifyContent: "center",
-          height: "30vh",
+          height: "10vh",
         }}
       >
-        <CreateStream
-          streamManagerAddress={"0xb5ED1eF2a90527b402Cd7e7d415027CB94E1Db4E"}
-          tokenAddress={"0xbc083D97825DA7f7182F37fcEc51818E196aF1FF"}
-          amountPerSecond={100000000000000}
-          reasonCode={"1"}
-          registerStream={(s: Stream) => console.log(s)}
-        />
+        {isProcessing && <p>Processing Transaction... </p>}
+        {isProcessed && <p>Transaction Successful!</p>}
+        {processTxError && <p>Error: {processTxError.message}</p>}
       </div>
-      <ul
+
+      <div
         style={{
           display: "flex",
           alignItems: "center",
           justifyContent: "center",
-          height: "20vh",
+          height: "30vh",
         }}
       >
-        <li>
-          <StreamStatus
-            stream={
-              new Stream(
-                "0xb5ED1eF2a90527b402Cd7e7d415027CB94E1Db4E",
-                "0x1C277bD41A276F87D3E92bccD50c7364aa2FFc69",
-                3,
-                usePublicClient(),
-                useWalletClient()?.data as WalletClient,
-              )
-            }
-          />
-        </li>
-      </ul>
+        <CreateStream
+          streamManagerAddress={"0xb5ED1eF2a90527b402Cd7e7d415027CB94E1Db4E"}
```

Comment:
been using the wrong address this whole time
```suggestion
          streamManagerAddress={"0x3543Faeeddb7bAbCbBB216B3627f9c5E0C39CE41"}
```